### DISCUSSION TITLE
Robotoff for the Folksonomy Engine

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-ai#369] [Q&A]
### Potential Use-Cases
- Predicting folksonomy engine properties that exist based on product data
- Proposing folksonomy engine properties that do not exist yet, based on product data
- Generate documentation for the property, have the user validate it, and after validation, create the property in the wiki if it does not exist

**Comment by baslia** [comment]
Hey, that's interesting.
Just to clarify after reading [this](https://wiki.openfoodfacts.org/Folksonomy_Engine), you would want a model to predict custom categories or attributes ?
